packages/amplify_authenticator/lib/src/widgets/social/social_button.dart

```diff
@@ -145,35 +145,36 @@ class _SocialSignInButtonState
 
   Widget get icon {
     final bool isDark = Theme.of(context).brightness == Brightness.dark;
-    switch (widget.provider) {
-      case AuthProvider.google:
-        return SocialIcons.googleLogo;
-      case AuthProvider.facebook:
-        return const Icon(
-          SocialIcons.facebook,
-          color: SocialIcons.facebookPrimaryColor,
-        );
-      case AuthProvider.amazon:
-        return const Icon(
-          SocialIcons.amazon,
-          color: SocialIcons.amazonPrimaryColor,
-        );
-      case AuthProvider.apple:
-        // This icon renders slightly off-center without this padding.
-        return Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: [
-            Icon(
-              SocialIcons.apple,
-              color: isDark ? Colors.white : Colors.black,
-            ),
-            const SizedBox(height: 4),
-          ],
-        );
-      default:
-        safePrint('Unsupported provider: ${widget.provider}');
-        return const SizedBox.shrink();
+    if (widget.provider == AuthProvider.google) {
+      return SocialIcons.googleLogo;
```

Comment:
Why is this not wrapped in `Icon` like the others?

Reply:
I don't remember lol - this was just reorganizing because you can no longer switch on AuthProvider